chart/templates/_helpers.yaml

```diff
@@ -233,6 +233,7 @@ If release name contains chart name it will be used as a full name.
       value: "false"
     {{- end }}
     {{ else if .Values.dags.gitSync.credentialsSecret }}
+    {{- if hasPrefix .Values.images.gitSync.tag "v3" }}
```

Comment:
if it's v3 it should add `GIT_SYNC_USERNAME` and `GIT_SYNC_PASSWORD` here you add `GIT_SYNC_USERNAME` and `GITSYNC_USERNAME`

Reply:
Thanks, I made a copy-paste error. Will fix it right away!